### PR TITLE
added cards for WZG sample

### PR DESCRIPTION
**Pull request by bth5032**
SUS WZG cards

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+
+import model loop_sm
+
+define p = g u c d s u~ c~ d~ s~
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat]: you can remove the proton and jet definitions, cfr https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by bth5032** [comment]
Hi, should be done.

**Comment by bth5032** [comment]
Just to be sure, removing the jet and proton definitions is just cosmetic, I do not need to rerun the gridpacks, correct?

**Comment by perrozzi** [comment]
a good practice is not to produce gridpacks until the cards are approved...

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.3.2.2               2015-09-06         *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat]: please remove this banned as the version is not anymore 2.3.2.2

**Comment by bth5032** [comment]
done

**Pull request merged**